crates/holochain/tests/tests/countersigning.rs

```diff
@@ -407,7 +407,7 @@ async fn alice_can_recover_from_a_session_timeout() {
 
     // Make sure the conductors are gossiping before creating posts
     conductors[0]
-        .require_initial_gossip_activity_for_cell(alice, 3, Duration::from_secs(30))
+        .require_initial_gossip_activity_for_cell(alice, 2, Duration::from_secs(30))
```

Comment:
This is now counting the number of *other* peers, rather than including ourselves